showcase/src/app/demo/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { StarkBreadcrumbConfig } from "@nationalbankbelgium/stark-ui";
+const componentName: string = "StarkBreadcrumb";
+
+/**
+ * Demo Breadcrumb component page
+ */
+@Component({
+	selector: "breadcrumb-component",
```

Comment:
The selector of a component should not contain "component" word.
Could you please change it for "breadcrumb" or change the component name for DemoBreadcrumbComponent and change the selector for "demo-breadcrumb"